scripts/changelog.js

```diff
@@ -8,7 +8,7 @@ const octokit = new Octokit();
 run();
 
 async function run() {
-  let packages = JSON.parse(exec('yarn workspaces info --json').toString().split('\n').slice(1, -2).join('\n'));
+  let packages = exec('yarn workspaces list --json').toString().split(require('os').EOL).filter(Boolean).map((x) => JSON.parse(x));
```

Comment:
maybe there's a better way to do this but basically `yarn workspaces info --json` doesn't exist after we upgraded to yarn 4 hence the change

Reply:
for now it's fine, though eventually we should turn it into a yarn plugin, then we don't have to do an exec call at all, we can use their workspaces API

we had to update it here as well when we switched to yarn 4